contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -72,7 +72,6 @@ left join tt_internal tt on t.oid = tt.typrelid
 where tt.typrelid is null
 and (t.relkind = 'r' or t.relkind = 'p')
 and t.relispartition = false
-and has_schema_privilege(t.relnamespace, 'USAGE')
```

Comment:
There are still few occurences like sys.sp_tables_view, sys.sp_special_columns_view, etc. where you missed to remove the `has_schema_privilege(USAGE)` redundant check. Please make sure we remove it from everywhere.

Reply:
Removed.